src/main/scala/wdl4s/AstTools.scala

```diff
@@ -286,6 +286,26 @@ object AstTools {
 
   def terminalMap(ast: Ast, source: WdlSource) = (findTerminals(ast) map {(_, source)}).toMap
 
+  def wdlSectionToStringMap(ast: Ast, node: String, wdlSyntaxErrorFormatter: WdlSyntaxErrorFormatter): Map[String, String] = {
+    ast.findAsts(node) match {
+      case a if a.isEmpty => Map.empty[String, String]
+      case a if a.size == 1 =>
+        // Yes, even 'meta {}' and 'parameter_meta {}' sections have RuntimeAttribute ASTs.
+        // In hindsight, this was a poor name for the AST.
```

Comment:
Discussed earlier in slack with @mcovarr - my concern was that the original author of that comment is someone I would have expected to fix that if it wasn't a big deal. I can see reasons why he wouldn't have that don't fit that model however.